sdks/python/apache_beam/examples/snippets/transforms/elementwise/filter_test.py

```diff
@@ -31,31 +31,26 @@
 
 
 def check_perennials(actual):
-  # [START perennials]
-  perennials = [
-      {'icon': '🍓', 'name': 'Strawberry', 'duration': 'perennial'},
-      {'icon': '🍆', 'name': 'Eggplant', 'duration': 'perennial'},
-      {'icon': '🥔', 'name': 'Potato', 'duration': 'perennial'},
-  ]
-  # [END perennials]
-  assert_that(actual, equal_to(perennials))
+  expected = '''[START perennials]
+{'icon': '🍓', 'name': 'Strawberry', 'duration': 'perennial'}
+{'icon': '🍆', 'name': 'Eggplant', 'duration': 'perennial'}
+{'icon': '🥔', 'name': 'Potato', 'duration': 'perennial'}
+[END perennials]'''.splitlines()[1:-1]
+  assert_that(actual, equal_to(expected))
 
 
 def check_valid_plants(actual):
-  # [START valid_plants]
-  valid_plants = [
-      {'icon': '🍓', 'name': 'Strawberry', 'duration': 'perennial'},
-      {'icon': '🥕', 'name': 'Carrot', 'duration': 'biennial'},
-      {'icon': '🍆', 'name': 'Eggplant', 'duration': 'perennial'},
-      {'icon': '🍅', 'name': 'Tomato', 'duration': 'annual'},
-  ]
-  # [END valid_plants]
-  assert_that(actual, equal_to(valid_plants))
+  expected = '''[START valid_plants]
+{'icon': '🍓', 'name': 'Strawberry', 'duration': 'perennial'}
+{'icon': '🥕', 'name': 'Carrot', 'duration': 'biennial'}
+{'icon': '🍆', 'name': 'Eggplant', 'duration': 'perennial'}
+{'icon': '🍅', 'name': 'Tomato', 'duration': 'annual'}
+[END valid_plants]'''.splitlines()[1:-1]
+  assert_that(actual, equal_to(expected))
 
 
 @mock.patch('apache_beam.Pipeline', TestPipeline)
-# pylint: disable=line-too-long
-@mock.patch('apache_beam.examples.snippets.transforms.elementwise.filter.print', lambda elem: elem)
+@mock.patch('apache_beam.examples.snippets.transforms.elementwise.filter.print', str)
```

Comment:
Do you need mock.patch() to be in a single line?

Reply:
The mocked function signature can get pretty long, and it's easier to have in a single line rather than having to split it into multiple lines. Since it's a fairly repetitive line that is not adding too much to the logic, I would prefer them to be in a single line. I added an exception for `@mock.patch` in the `.pylintrc` so we don't have to explicitly state the `pylint: disable`.

Reply:
I do not believe we should make an exception on disabling lint on these.

Reply:
Some mock.patch lines could be broken into multiple lines, but that would make them inconsistent. Then there are others that are so long that the function path string does not even fit in the 80 characters, so we have to do the disable/enable on pylint (or concatenating strings but that's even uglier).

For consistency, I would like to keep all of them in a single line, and I also think it's easier to read like that.

If you prefer the explicit `# pylint: disable`/`# pylint: enable` in every file for the `@mock.patch` lines specifically, I can add them back and revert the changes in the `.pylintrc`.

Reply:
I would prefer something like

```
@mock.patch(
  'apache_beam.examples.snippets.transforms.elementwise.filter.print',
  lambda elem: elem)
```

Reply:
Alright, I can make that